ext/B/t/optree_specials.t

```diff
@@ -45,52 +45,54 @@ checkOptree ( name	=> 'BEGIN',
 	      prog	=> $src,
 	      strip_open_hints => 1,
 	      expect	=> <<'EOT_EOT', expect_nt => <<'EONT_EONT');
-# -     <@> lineseq KP ->7
```

Comment:
Are the changes to this file intentional?

Reply:
They are required. t/optree_specials.t uses OptreeCheck.pm to run its tests. Among several other modules, it uses Exporter. Any time any of those modules change, it affects the BEGIN blocks and so makes these tests change. 

I considered fixing the jitter by altering OptreeCheck.pm but decided it was WAY out of scope to do so here. That would actually cause more jitter.